rules/mentoring-programs/rule.md

```diff
@@ -60,39 +62,43 @@ Here are some of the things you can learn:
 
 ## Scheduled mentoring - How does it work?
 
-As a mentee, you will be driving your progress. If you need additional motivation, add what you are learning to your Teams Status. Mentors - you can do the same, let people know what you're mentoring.
+The first step is for mentees to find mentors or vice-versa. 
 
-![Figure: Let your colleagues know what knowledge you are sharing and learning... subtly 🙂](mentoring-teams-status.png)
+![Figure: A mentee reaches out to someone they know has great skills in a particular area, and they ask to be mentored](mentoring-talk-smaller.png)
 
-You will need to book a few appointments with your mentor:
+Once a pairing has been established, the mentor is responsible for scheduling a few meetings with the mentee. 
 
-* 5 minute chats at the end of every week to catch up to see how you're progressing 
-* 3x monthly mentoring sessions (about 30mins) to work on your target areas.
+1. Initial Meeting (30 mins): Introduction to the program and determination of the mentee's specific learning targets.
+2. Weekly Check-ins (5 mins): A quick chat at the week's end to gauge the mentee's progress.
+3. Monthly Sessions (for 3 months, 30 mins each): Intense mentoring sessions to tackle mentee's chosen target areas.
+4. Final Meeting (30 mins): Wrap up, retrospection, and gathering of feedback to refine the program further.
 
-It is motivating to know that you will be checking-in with your mentor regularly. At the end of your 3 months, sit down and recap how well it went for you. It's helpful to give a mark out of 10. 
+As a mentee, add what you are learning to your Teams Status. Mentors - you can do the same, let people know what you're mentoring.
 
-![Figure: A mentee reaches out to someone they know has great skills in a particular area, and they ask to be mentored](mentoring-talk-smaller.png)
+![Figure: Let your colleagues know what knowledge you are sharing and learning... subtly 🙂](mentoring-teams-status.png)
 
-### 1st Meeting
+### Initial Meeting
 
 * Tell the mentor what you would like to achieve by the end of the 3 months
 * Work out your goals & set targets
 * Then the mentee works independently for a month, checking-in when needed
 * At the end of each week, check-in for 5 mins to let them know how you're going
 
-### 2nd Meeting
+### 3 Monthly Meetings
```

Comment:
Hi @divgul 

1. Should this be "3-Month" instead of "3 Monthly"

Reply:
Hi @tiagov8, 

> 1. Should this be "3-Month" instead of "3 Monthly"

"3 Monthly" is correct as there are 3 meetings, one per month